.github/actions/docker-build-and-push/action.yaml

```diff
@@ -77,7 +88,7 @@ runs:
           suffix=-devel${{ inputs.tag-suffix }}
 
     - name: Docker meta for runtime
-      if: ${{ github.event_name == 'workflow_dispatch' }} || ${{ (github.event_name == 'push' && github.ref_type == 'tag') }}
+      if: ${{ github.event_name == 'workflow_dispatch' || (github.event_name == 'push' && github.ref_type == 'tag') }}
```

Comment:
I'm surprised this worked before, the correct syntax is like [this](https://docs.github.com/en/actions/learn-github-actions/expressions#example). `( )` is enough for local grouping.